packages/blade-old/src/icons/Direction/Direction.native.js

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Svg, { Path } from 'react-native-svg';
+import { getThemeColors } from '../../_helpers/theme';
+import colors from '../../tokens/colors';
+
+export default function Direction({ width, height, fill }) {
```

Comment:
since we are not spreading props when you apply flex and space on this Icon it'll break. This was raised by @anuraghazra here https://github.com/razorpay/blade/pull/367#discussion_r759962874